bazaarvoice/astyanax
# PD-195972: the SonarQube integration removing

## Jira Issue
[PD-195972](https://bazaarvoice.atlassian.net/browse/PD-195972)

## What Are We Doing Here?
Remove the SonarQube integration

## How to Test and Verify

1. Check out this PR
2. Run: `mvn clean install -U -Ddependency-check.skip=true`
3. Run: `mvn -e -Ddependency-check.skip=true verify`
4. Check if [the Jenkins job](https://jenkins.nexus.bazaarvoice.com/job/cassandra/job/astyanax-pipeline/view/change-requests/job/PR-26/) builds successfully

## Risk Level
`Low`

## Required Testing
`Smoke`
